packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGImage/SVGImage.js

```diff
@@ -0,0 +1,54 @@
+import { PropTypes } from 'prop-types'
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+
+import { draggable } from '@plugins/drawingTools/components'
+
+export const DraggableImage = styled(draggable('image'))`
+  cursor: move;
+`
+
+const INVERT =
+  `<svg style="position: fixed; right: 100%; top: 100%; visibility: hidden;">
+    <defs>
+      <filter id="svg-invert-filter" color-interpolation-filters="sRGB">
+        <feComponentTransfer>
+          <feFuncR type="table" tableValues="1 0"/>
+          <feFuncG type="table" tableValues="1 0"/>
+          <feFuncB type="table" tableValues="1 0"/>
+        </feComponentTransfer>
+      </filter>
+    </defs>
+  </svg>`
+
+export default function SVGImage ({
+  invert,
+  move,
+  naturalHeight,
+  naturalWidth,
+  onDrag,
+  src,
+  subjectID
+}) {
+  useEffect(function onInvert () {
+    if (invert) {
+      if (!document.getElementById('svg-invert-filter')) {
+        document.body.insertAdjacentHTML('afterbegin', INVERT)
+      }
+    }
+  }, [invert])
+
+  const SubjectImage = move ? DraggableImage : 'image'
+
+  return (
+    <SubjectImage
+      aria-label={`Subject ${subjectID}`}
+      dragMove={move ? onDrag : () => null}
+      height={naturalHeight}
+      role='img'
+      style={invert ? { filter: 'url("#svg-invert-filter")' } : {}}
```

Comment:
```suggestion
      filter={invert ? 'url("#svg-invert-filter")' : undefined}
```
I think this is the correct syntax if you want this component to render SVG, rather than SVG/HTML. It depends whether you envisage this component ever being used in a pure SVG document.

Reply:
I've tested this in Safari and it works.

Reply:
I've tried this out in Safari, and it works.